telemeter/telemeter.py

```diff
@@ -68,6 +68,10 @@ def from_json(cls, data: dict):
         peak_usage = data["totalusage"].get("peak", 0)
         offpeak_usage = data["totalusage"].get("offpeak", 0)
 
+        total_usage = peak_usage + offpeak_usage
+        if not total_usage:
+            total_usage = data.get('totalusage', {}).get('includedvolume', 0)
```

Comment:
Looks good! If you could just switch to double quotes here, to please my OCD :)

Reply:
sure, no problem!